src/functionaltest/java/com/ericsson/ei/config/Utils.java

```diff
@@ -0,0 +1,117 @@
+package com.ericsson.ei.config;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.Properties;
+
+import org.apache.commons.compress.archivers.tar.TarArchiveEntry;
+import org.apache.commons.compress.archivers.tar.TarArchiveInputStream;
+import org.apache.commons.compress.compressors.bzip2.BZip2CompressorInputStream;
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.io.IOUtils;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Utils {
+    private static final Logger LOGGER = LoggerFactory.getLogger(Utils.class);
+
+    /**
+     * Loads properties from file path.
+     *
+     * @param filePath
+     *            path to the properties file
+     * @return properties object
+     */
+    public static Properties getProperties(final String filePath) {
+        Properties properties = new Properties();
+        File file = new File(filePath);
+        try {
+            properties.load(new FileInputStream(file));
+        } catch (IOException e) {
+            LOGGER.error("Failed load properties from path.\nError: {}", e.getMessage());
+        }
+        return properties;
+    }
+
+    /**
+     * Extracts a BZip2 archive to a destination folder.
+     *
+     * @param firefoxTarballFilePath
+     *            string containing the firefox BZip2 filepath.
+     * @param destination
+     *            string containing a destination path.
+     */
+    public static void extractBZip2InDir(final String firefoxBZip2FilePath, String destinationPath) {
+        LOGGER.info("Extracting firefox BZip2 archive...");
+        try (TarArchiveInputStream fileInput = new TarArchiveInputStream(
```

Comment:
Generally you do not want 2 try catch inside each other.

You may catch different errors in the same try catch, either by doing something like this
try {
} catch (someError || AnotherError) {
}
I didn´t get that one to work though, maybe I wrote wrong errors :)
Or like this.
try {
} catch (and error) {
} catch (Another erro5r) {
}

Reply:
I see you already do the later onbe so the inner try catch is not nessesary.

Reply:
I took this code from another component but i agree

Reply:
I took this code from another component and i guess the reason why it's made this way is that the fileInput needs to be closed after exceptions thrown within. Although apperently in java 7 and onwards try-catches with resources will automatically be closed. Therefore the fileInput.close() should not even be needed.  https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html